PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-vm/src/main/java/com/microsoft/azure/toolkit/intellij/vm/ssh/CommunityConnectBySshAction.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.azure.toolkit.intellij.vm.ssh;
+
+import com.intellij.openapi.project.Project;
+import com.microsoft.azure.toolkit.lib.common.messager.AzureMessager;
+import com.microsoft.azure.toolkit.lib.common.operation.AzureOperation;
+import com.microsoft.azure.toolkit.lib.common.task.AzureTaskManager;
+import com.microsoft.azure.toolkit.lib.compute.virtualmachine.VirtualMachine;
+import org.jetbrains.plugins.terminal.ShellTerminalWidget;
+import org.jetbrains.plugins.terminal.TerminalView;
+
+import javax.annotation.Nonnull;
+import java.io.IOException;
+import java.nio.file.Paths;
+
+public class CommunityConnectBySshAction {
+    private static final String SSH_TERMINAL_TABLE_NAME = "SSH - %s";
+    private static final String CMD_SSH_KEY_PAIR = "ssh %s@%s -i %s";
```

Comment:
no need to set cert path which will be used by ssh by default

Reply:
done